tests/saitests/py3/sai_qos_tests.py

```diff
@@ -917,54 +915,10 @@ def runTest(self):
-            # dscp  6 -> queue 1           queue 1         queue 6                                         queue 1                                  queue 6                     # noqa E501
-            # rest 56 dscps -> queue 1
-            # So for the 64 pkts sent the mapping should be the following:
-            # queue 1    56 + 2 = 58       56 + 3 = 59     56 + 1 = 57                                     59                                        57                         # noqa E501
-            # queue 2/6  1                 0               1                                                0                                         0                         # noqa E501
-            # queue 3/4  1                 1               1                                                1                                         1                         # noqa E501
-            # queue 5    1                 1               1                                                1                                         1                         # noqa E501
-            # queue 7    0                 1               1                                                1                                         1                         # noqa E501
-            assert (queue_results[QUEUE_0] == 1 + queue_results_base[QUEUE_0])
-            assert (queue_results[QUEUE_3] == 1 + queue_results_base[QUEUE_3])
-            assert (queue_results[QUEUE_4] == 1 + queue_results_base[QUEUE_4])
-            assert (queue_results[QUEUE_5] == 1 + queue_results_base[QUEUE_5])
-            if dual_tor or (dual_tor_scenario is False) or (leaf_downstream is False):
-                assert (queue_results[QUEUE_2] == 1 +
-                        queue_results_base[QUEUE_2])
-                assert (queue_results[QUEUE_6] == 1 +
-                        queue_results_base[QUEUE_6])
-            else:
-                assert (queue_results[QUEUE_2] == queue_results_base[QUEUE_2])
-                assert (queue_results[QUEUE_6] == queue_results_base[QUEUE_6])
-            if dual_tor_scenario:
-                if (dual_tor is False) or leaf_downstream:
-                    assert (queue_results[QUEUE_1] ==
-                            59 + queue_results_base[QUEUE_1])
-                else:
-                    assert (queue_results[QUEUE_1] ==
-                            57 + queue_results_base[QUEUE_1])
-                # LAG ports can have LACP packets on queue 7, hence using >= comparison
-                assert (queue_results[QUEUE_7] >= 1 +
-                        queue_results_base[QUEUE_7])
-            else:
-                assert (queue_results[QUEUE_1] == 58 +
-                        queue_results_base[QUEUE_1])
-                # LAG ports can have LACP packets on queue 7, hence using >= comparison
-                assert (queue_results[QUEUE_7] >= queue_results_base[QUEUE_7])
+
+            for tc in range(7):
```

Comment:
run local test, observed 3 regressions on 8111/8102/8101:
testQosSaiLosslessVoq, testQosSaiSeparatedDscpQueueMapping and testQosSaiDwrr / testQosSaiDwrrWeightChange

upload test log to jira-617, please help to analyze

Reply:
For testQosSaiLosslessVoq failure on 8111, compared the failed log and passed log, they used the same qos params, didn't get clue from log. If the failure is persistent, we can schedule a debug session to see why PFC is not triggered. Cisco local run passed.

For testQosSaiSeparatedDscpQueueMapping and testQosSaiDwrr[WeightChange], they failed with T1 downstream redundancy topology. Updated PR fixed this. Verified on 8101 T1 DownstreamRedundancyTypes Gemini.